fault-tolerance/fault-tolerance/src/main/java/io/helidon/faulttolerance/Bulkhead.java

```diff
@@ -32,6 +32,32 @@
  */
 @RuntimeType.PrototypedBy(BulkheadConfig.class)
 public interface Bulkhead extends FtHandler, RuntimeType.Api<BulkheadConfig> {
+
+    /**
+     * Counter for all the calls in a bulkhead.
+     */
+    String FT_BULKHEAD_CALLS_TOTAL = "ft.bulkhead.calls.total";
```

Comment:
If we remove the `.total` suffix from the value then probably remove it from the name of the constant as well. (I won't repeat the same note for the other counters but it would apply to them too.)

Reply:
See above